public/js/utils/source-map.js

```diff
@@ -5,9 +5,17 @@ import type { Location } from "./actions/types";
 const { getSource, getSourceByURL } = require("../selectors");
 const { isEnabled, getValue } = require("../feature");
 
-const sourceMapWorker = new Worker(
-  getValue("baseWorkerURL") + "source-map-worker.js"
-);
+let sourceMapWorker;
+function restartWorker() {
+  if(sourceMapWorker) {
+    sourceMapWorker.terminate();
+  }
+  sourceMapWorker = new Worker(
+    getValue("baseWorkerURL") + "source-map-worker.js"
+  );
+}
+restartWorker();
```

Comment:
Yeah, I use `afterEach` to terminate it. That's a good point that we could do this automatically though. We'll need to restart all of the workers, which I assume would be ok... I'll try it out!